changelog/4830.feature.rst

```diff
@@ -0,0 +1,17 @@
+Added conversation sessions to trackers. A conversation session represents the
```

Comment:
Let's try to follow the same convention as for docstrings: first-line summary, line break and then details. (makes the changelog easier to digest)